EventFilter/EcalRawToDigi/src/MatacqRawEvent.cc

```diff
@@ -80,10 +80,9 @@ const MatacqRawEvent::field32spec_t MatacqRawEvent::side32              = {13,0x
 
 void MatacqRawEvent::setRawData(const unsigned char* pData, size_t maxSize){
   error = 0;
-  unsigned char* begin = (unsigned char*) pData;
-  int16le_t* begin16 = (int16le_t*) pData;
+  const int16le_t *begin16 = (const int16le_t *) pData;
   uint32le_t* begin32 = (uint32le_t*) pData;
```

Comment:
The static analzyer says `begin32` needs to be made `const`:
`const uint32le_t *begin32 = (const uint32le_t *) pData;`
But then you will need to change EcalRawToDigi//interface/MatacqRawEvent.h. In that file change line 367 to:
` static int read32(const uint32le_t *pData, field32spec_t spec, bool ovfTrans = false);`
and change line 404 to:
` const uint32le_t *daqHeader;`